panther_analysis_tool/util.py

```diff
@@ -252,3 +253,31 @@ def log_and_write_to_file(msgs: List[str], filename: TextIO) -> None:
     for msg in msgs:
         filename.write(msg + "\n")
         logging.info(msg)
+
+
+def lookup_analysis_id(analysis_spec: Any) -> str:
+    analysis_type = analysis_spec.get("AnalysisType")
```

Comment:
good change 😄 
unit tests would be cool to introduce here.

Reply:
Added some unit tests